upgrade/1.2/Stage_2.md

```diff
@@ -125,21 +125,29 @@ upgrade procedure pivots to use `ncn-m002` as the new "stable node", in order to
 
 ## Stage 2.4
 
+Apply the hotfix for `kdump`:
+
+```bash
+ncn-m002# /usr/share/doc/csm/scripts/hotfixes/kdump/hotfix.sh
+```
+
```

Comment:
I think expected output should be added here as well.

Reply:
oh I thought I did this, I pasted it three times.. wonder where I put it then.

Reply:
oh the hotfix readme, deploy ncns, and deploy final ncn, that's where.